src/api-reference/travel-profile/02-form-payment-resource.markdown

```diff
@@ -87,33 +87,33 @@ Name | Type | Format | Description |
 
 The CreditCard element contains the following child elements:
 
-Name | Type | Format | Description |
+Name | Type | Required | Description |
 ------------|-----------------|---------|-------------|
-`Vendor` |`string` |`-` |The card vendor. One of the following options: Unknown, AmericanExpress, DinersClub, Discover, MasterCard, Visa, CarteBlanche, Enroute, UniversalAirTravel, JCB, AmericanAirlines, DeltaEquity, NorthwestAirlines, TWAGetaway, UnitedTravelCard, UnitedCreditCard, EuroCard, CanadianAirlines, AlaskaAirBarter, PurchaseOrder, AwardCredit, Debit, ChinaUnionPay, Cash, CompanyPaid, CreditCard |
-`AccountNo` | `string`|`-` |The credit card account number. |
+`Vendor` |`string` |`Creation` |The card vendor. One of the following options: Unknown, AmericanExpress, DinersClub, Discover, MasterCard, Visa, CarteBlanche, Enroute, UniversalAirTravel, JCB, AmericanAirlines, DeltaEquity, NorthwestAirlines, TWAGetaway, UnitedTravelCard, UnitedCreditCard, EuroCard, CanadianAirlines, AlaskaAirBarter, PurchaseOrder, AwardCredit, Debit, ChinaUnionPay, Cash, CompanyPaid, CreditCard |
```

Comment:
Same feedback as above -- let's break out this long list of options into a separate table which is referenced.

Reply:
Already have this reference table, good call on this.